androidApp/src/main/java/com/mbta/tid/mbta_app/android/pages/NearbyTransitPage.kt

```diff
@@ -112,6 +116,10 @@ fun NearbyTransitPage(
         vehiclesRepository.disconnect()
     }
 
+    LaunchedEffect(mapViewModel.lastMapboxErrorTimestamp.collectAsState(initial = null).value) {
+        mapViewModel.loadConfig()
+    }
```

Comment:
I'm not sure how to write a test that covers this - looking more closely at existing tests to see if there is an analogous case already

Reply:
Thanks for the suggestion! It ended up being a little hairy to mock at that deeper level, but it got me thinking about mocking at the view model level which worked!